tests/integration/page_elements.py

```diff
@@ -0,0 +1,324 @@
+from lazy.lazy import lazy
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.support.select import Select
+from group_project_v2.components.stage import StageState
+
+__author__ = 'e.kolpakov'
+
+
+class BaseElement(object):
+    def __init__(self, browser, element):
+        self._browser = browser
+        self._element = element
+
+    @property
+    def browser(self):
+        return self._browser
+
+    @property
+    def element(self):
+        return self._element
+
+    def __getattr__(self, item):
+        if hasattr(self.__dict__, item):
+            return getattr(self.__dict__, item)
+        if hasattr(self.element, item):
+            return getattr(self.element, item)
+        return super(BaseElement, self).__getattr__(item)
+
+    def make_element(self, dom_element, element_type):
+        return element_type(self.browser, dom_element)
+
+
+    def make_elements(self, css_selector, element_type):
+        elements = self.element.find_elements_by_css_selector(css_selector)
```

Comment:
I feel like we're re-inventing the wheel a bit here. Isn't bok_choy available to the XBlock SDK now? Why not use bok_choy pages and queries and whatnot?